Ribbon.cs

```diff
@@ -52,12 +52,31 @@ public string GetCustomUI(string ribbonID)
         {
             try
             {
+                if (ribbonID != "Microsoft.Outlook.Mail.Read" && ribbonID != "Microsoft.Outlook.Explorer")
+                {
+                    return "";
+                }
                 Logger.Log("Start to setup custom UI");
                 string ribbonTemplate = GetResourceText("TypicalReply.Ribbon.xml");
                 var xmlDocument = new XmlDocument();
                 xmlDocument.LoadXml(ribbonTemplate);
                 string namespaceURI = xmlDocument.ChildNodes[1].NamespaceURI;
                 RuntimeParams global = RuntimeParams.GetInstance();
+
+                XmlNode removeTargetNode = null;
+                if (ribbonID == "Microsoft.Outlook.Mail.Read")
+                {
+                    // An error "TabMail does not found" can be raised if TabMail is specified 
+                    // when target ribbon is not "Microsoft.Outlook.Mail.Read"
+                    removeTargetNode = xmlDocument.SelectSingleNode("//*[@idMso='TabMail']");

```

Comment:
Currently we remove the node from loaded XML..
As another approach, we can switch the Xml document to be loaded depending on `ribbonID`.